source/index.html.md

```diff
@@ -35,7 +40,7 @@ This update was for documentation only, nothing changes in the consumer side.
 ### 2021-06-24
 
 Fixed documentation inconsistency between REST Api service and the implementation:
-* in user_trades service documentation, now 'book' is required.
+
```

Comment:
Does this fix the formatting problems mentioned above?

Reply:
yes